truss/contexts/image_builder/serving_image_builder.py

```diff
@@ -27,6 +26,10 @@
     TEMPLATES_DIR,
     TRITON_SERVER_CODE_DIR,
 )
+from truss.contexts.image_builder.cache_warmer import (
```

Comment:
it it a good idea to import from cache_warmer? Not blocking but just curious.

Reply:
yeah. It's fine in this case since the context builder includes all of these files and they importable. The trouble is the other way around, when you want to use a file in the docker build itself that you didn't copy there previously.